### DISCUSSION TITLE
Escaping HTML tags with backtick in JSDoc

### DISCUSSION THREAD

**Discussion by RBilly** [adobe/react-spectrum#6026] [Ideas]
Hi,
First of all, thank you to all maintainers of this awesome project ! 
I'm starting to build my own UI components on top of RAC. 
I rely on Storybook autodocs do generate the documentation. 
It uses the RAC JSDoc on props type to generate a description and renders it using markdown. 
It works very well except in a specific case:
When the JSDoc contains unescaped HTML tags, markdown tries to render the HTML element which causes error in some cases.

For exemple, `form` property of ButtonProps:
```
 /**
   * The <form> element to associate the button with.
   * The value of this attribute must be the id of a <form> in the same document.
   */
  form?: string,
```
This throws an error because it can't render a `<form>` inside a `<p>` element but we don't want to actually render an html `<form>`.

I'm suggesting to improve the JSDoc by escaping the HTML tags in JSDoc with backticks.

Actually, since the issue is also present here:
https://react-spectrum.adobe.com/react-aria-starter/index.html?path=/docs/button--docs
Maybe I should open an issue ?

Thank you again for your amazing work

**Comment by snowystinger** [comment]
Closing as issue was opened
https://github.com/adobe/react-spectrum/issues/6027
